doc/starting.rst

```diff
@@ -80,6 +80,22 @@ connection::
     pudb:6899: Please telnet into 127.0.0.1 6899.
     pudb:6899: Waiting for client...
 
+Debugging forked processes
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On Unix system, e.g. Linux & MacOS e.g. Linux & MacOS, debugging a forked
+process is supported::
```

Comment:
When running some parts of the code in a different process, e.g. via python's multiprocessing, the tty is not attached anymore. This leads to errors for pdb and standard pudb. E.g. consider this `script.py`:
```
from multiprocessing import Process
def f(name):
    breakpoint()
    print('hello', name)

p = Process(target=f, args=('bob',))
p.start()
p.join()
```

Running `PYTHONBREAKPOINT=pudb.set_trace python script.py` results in `TypeError: ord() expected a character, but string of length 0 found` due to the missing tty. However, using `PYTHONBREAKPOINT=pudb.forked.set_trace python script.py` works in this case, since the input and output are attached explicitly. Using the `remote` pudb version would also work, but requires you to telnet into the debugger, whereas this solution directly opens the debugger on the commandline used to start python.

Reply:
Just to highlight that this seems to be a use-case for more people:
There's a [medium article](https://medium.com/@auro_227/use-pudb-to-debug-python-multiprocessing-code-c0c5551d010c) about exactly this use-case, proposing `pudb.remote`.
Also a [stackoverflow question](https://stackoverflow.com/questions/47967490/cannot-use-pudb-for-multiprocessing), and an old issue (https://github.com/inducer/pudb/issues/286).

And even more discussion when looking for any python debugger that is able to debug forked processes, e.g. [here](https://stackoverflow.com/questions/4716533/how-to-attach-debugger-to-a-python-subproccess/23654936#23654936)

Reply:
TIL that this was a problem. That (and your general approach) make sense to me now. It would be fantastic if you could improve the docs a bit to make it a bit more obvious what problem is being solved, for example by incorporating the example you supplied. Also, I'd suggest using the subsection title "Using the debugger after forking" and describing that stdin/stdout are not usually available then.

Reply:
(And thanks for taking the time to explain!)

Reply:
Sure thing :-) I tried to incorporate the example. Feel free to directly add suggestions if you see more room for improvement :+1: